sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -221,6 +243,8 @@ public static Write write() {
 
       abstract Builder setTableId(String tableId);
 
+      abstract Builder setLocalhost(String host);
```

Comment:
Users will provide and configure their `CloudBigtableOptions`. Given that, it would be error prone design to either mutate the one they give us or, potentially, let them override these settings if they build in the wrong order.

Note that if I call `withLocalhost()` and then `withBigtableOptions` I lose the localhost config. This is not a good scenario.

Cloud Bigtable options itself may -- and in fact based on history is likely to -- change its configuration scheme in a way that makes it difference, harder, or even impossible for us to set localhost this way. It may be parameterized completely differently in the future. (Indeed, they may remove some of the redundant-seeming configuration where you have to set three different endpoints to use localhost.)

It's important for our transform to have one and only one place to configure these things, and given that we already expose the CloudBigtableOptions object, that's where it should be.

(I am judging BigtableIO by BigtableIO alone; so DatastoreIO is not especially relevant. Yet, I don't see the parallel -- DatastoreIO does not expose an options class and FWICT the localhost options is not configurable another way.)